docs/support/glossary.md

```diff
@@ -0,0 +1,255 @@
+# Glossary
+
+<style>
+  dd {
+    border-left: 5px solid cadetblue;
+    padding-left: 10px;
+  }
+  dt {
+    color: cadetblue;
+    font-weight: bold;
+  }
+  a {
+    text-decoration: underline;
+  }
+</style>
+
+<html>
+<dl>
+  <dt>ABC Analysis class</dt>
+  <dd>This class system categorizes products based on their impact on cost. Products in Class A contribute to the majority of costs because you use and buy them most often. You buy Class B products more infrequently. Class C items are not high value and do not need many follow-ups.</dd>
+  
+  <br>
```

Comment:
You can remove the <br/> elements as we can add space using CSS.

Reply:
Done. You're right: in my typography notes, I wrote down long ago to never use carriage returns. Looks like I forgot this time. Good lesson to learn before making that mistake for a big company.

Reply:
@jmiranda Whoops didn't @ you. Anyway, it's done.